controllers/gce/backends/backends.go

```diff
@@ -116,20 +152,43 @@ func (b *Backends) create(igs []*compute.InstanceGroup, namedPort *compute.Named
 	if err != nil {
 		return nil, err
 	}
-	// Create a new backend
-	backend := &compute.BackendService{
-		Name:     name,
-		Protocol: "HTTP",
-		Backends: getBackendsForIGs(igs),
-		// Api expects one, means little to kubernetes.
-		HealthChecks: []string{hc.SelfLink},
-		Port:         namedPort.Port,
-		PortName:     namedPort.Name,
-	}
-	if err := b.cloud.CreateBackendService(backend); err != nil {
-		return nil, err
+	errs := []string{}
+	for _, bm := range []BalancingMode{Rate, Utilization} {
```

Comment:
You are basically first trying to set it to RATE mode right? If fail, then try to set on UTILIZATION mode.

Can you comment? it is not obvious what are you trying to do with the loop.